tests/integration/models/reddit/test_live.py

```diff
@@ -156,3 +156,14 @@ def test_close(self, _):
         with self.recorder.use_cassette(
                 'TestLiveThreadContribution_close'):
             thread.contrib.close()
+
+
+class TestLiveUpdateContribution(IntegrationTest):
+    @mock.patch('time.sleep', return_value=None)
```

Comment:
I see. Thanks for the explanation.

> Typically only tests with more than one API interaction involve a sleep. I always just try it without, and it'll raise an exception if a sleep falls through without being patched.

Although this is a good practice, some (cached?) responses doesn't have rate limit headers, so some tests pass without raising an exception in the manner.